js/src/views/Signer/components/TransactionMainDetails/transactionMainDetails.js

```diff
@@ -14,52 +14,43 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+import MapsLocalGasStation from 'material-ui/svg-icons/maps/local-gas-station';
```

Comment:
Nothing to do with this PR, but I was thinking about this:
What about have an Icons util somewhere in the UI ? We don't use that many icons, but still whenever I want to find the right one I end up going to design.google.com. There might even be some inconsistencies in the app (using different icons for the same purpose).